src/index.ts

```diff
@@ -36,21 +37,18 @@ export function optimize(
        *  array of numbers, callback, or array of callbacks. Each kind of shape has default parameters so it could be undefined
        */
       parameters?: {
```

Comment:
This does not make sense and I guess we don't really have testcases to define allowed range parameters.

There are 2 changes to do here. In this object parameters we should be able to define the range for all the allowed parameters, currently x, y, fwhm and mu. Those are the 'default' parameters.

We should also allow to set those parameters at the level of each peaks. I will create a new issue for this: https://github.com/mljs/spectra-fitting/issues/79